shoop/core/order_creator/_source.py

```diff
@@ -301,7 +355,16 @@ def _compute_shipping_method_lines(self):
             for line in self.shipping_method.get_source_lines(self):
                 yield line
 
-    def _get_product_lines(self):
+    def _add_lines_from_modifiers(self, lines):
+        """
+        Add lines from OrderSourceModifiers to given list of lines.
```

Comment:
(That still won't make it impossible for them to stick their grubby fingers into the lines themselves, of course.)